### ISSUE TITLE
Could not resolve SDK directory from [/../netcoreapp1.0/ubuntu.14.04-x64]

### ISSUE THREAD

**Issue by colemickens** [dotnet/cli#2134]
## Steps to reproduce
1. Use this Dockerfile: https://raw.githubusercontent.com/colemickens/polykube/dev/dotnet/dockerfiles/aspnet-base/Dockerfile

2. Use this code: https://github.com/colemickens/polykube/tree/f5910f5a6a616ca60f497800026c04a080852894/dotnet

3. `dotnet restore; dotnet run`

## Expected  behavior

It compiles and runs.

## Actual behavior

```
Project Polykube.Api (.NETCoreApp,Version=v1.0) was previously compiled. Skipping compilation.
Tracing enabled
--- Invoked host main = {
/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64/Polykube.Api
--additionalprobingpath
/root/.nuget/packages
}
Resolved fxr [/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64/libhostfxr.so]...
Tracing enabled
Own DLL path=[/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64/Polykube.dll]
Checking if CoreCLR path exists=[/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64/libcoreclr.so]
Host operating in Muxer mode
--- Executing in muxer mode...
--- Resolving dotnet from working dir
Probing path [/opt/polykube/Polykube.Api/global.json] for global.json
Probing path [/opt/polykube/global.json] for global.json
Probing path [/opt/global.json] for global.json
Terminating global.json search at []
--- Resolving SDK version from SDK dir [/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64/sdk]
No production version found, so using latest prerelease
Checking if resolved SDK dir [/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64/sdk/-1.-1.-1] exists
Resolved SDK dir is []
Found CLI SDK in: 
Could not resolve SDK directory from [/opt/polykube/Polykube.Api/bin/Debug/netcoreapp1.0/ubuntu.14.04-x64]
```

## Environment data
`dotnet --version` output: `1.0.0-beta-002111`

## Full log

https://gist.githubusercontent.com/colemickens/f1ad478f8729955b6f5678711ad32d0f/raw/d8b4b4cb97f54f68829240375fc60a225730f744/gistfile1.txt

**Comment by colemickens** [comment]
I'm assuming that I should use `netcoreapp1.0` with the `1.0.0-beta-002111` build. If not, I can revert to `netstandard1.3`, but then I get a different error.

**Comment by pakrym** [comment]
Dup of https://github.com/dotnet/cli/issues/2125

**Comment by colemickens** [comment]
This is fixed, at least I can't repro in `dotnet-dev-1.0.0-beta-002131` anymore.
